### DISCUSSION TITLE
Using next-auth in an iframe

### DISCUSSION THREAD

**Discussion by Inkapa** [nextauthjs/next-auth#4486] [Help]
I have a question about using next-auth with iframes and popups.

I'm using the `azure-ad` provider, but my app is running in an iframe so the app can't redirect to microsoft's login page. How could I solve this? 

Is there a way to open the login in a new tab or a popup instead of the current page? I've looked into `react-new-window` and the solution via https://github.com/nextauthjs/next-auth/issues/178#issuecomment-757513968 but it doesn't seem to work for me.
Not sure if there's a better/up-to-date solution?

**Comment by ricardasjak** [comment]
I am having same problem.
